src/flowMC/strategy/parallel_tempering.py

```
@@ -412,14 +415,14 @@ def _exchange(
     def _adapt_temperature(
         self,
         temperatures: Float[Array, " n_temps"],
-        do_accept: Int[Array, "n_chains n_temps 1"],
+        do_accept: Bool[Array, "n_chains n_temps 1"],
```

Ruff (0.16.0)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)